src/extractor/MappingExtractor.ts

```diff
@@ -10,68 +14,75 @@ export class MappingExtractor {
     elements: ProcessableElementDefinition[],
     fhir: fhirdefs.FHIRDefinitions
   ): ExportableMapping[] {
-    const mappings =
-      input.mapping?.map(m => {
-        const mapping = new ExportableMapping(m.identity);
-        mapping.source = input.name;
-        if (m.name) mapping.title = m.name;
-        if (m.uri) mapping.target = m.uri;
-        if (m.comment) mapping.description = m.comment;
-        return mapping;
-      }) ?? [];
-    elements.forEach(element => {
-      this.extractRules(element, mappings);
-    });
+    const mappings = this.extractMappings(input, elements);
 
     // Filter out mappings on the parent - only include mappings new to the profile
     // or mappings on the base definition with additional mapping rules
+    // TODO: Look for parents from the local package being processed. This requires a MasterFisher.
     const parent = fhir.fishForFHIR(
       input.baseDefinition,
       utils.Type.Resource,
       utils.Type.Type,
       utils.Type.Profile,
       utils.Type.Extension
     );
-    const newItems = differenceWith(input.mapping, parent?.mapping, isEqual);
-    const newMappings = mappings.filter(mapping => newItems.some(i => i.identity === mapping.name));
-    const inheritedMappings = mappings.filter(mapping =>
-      parent?.mapping.some((i: fhirtypes.StructureDefinitionMapping) => i.identity === mapping.name)
-    );
 
-    // For each inherited mapping, look to see if there are any new rules
-    const changedInheritedMappings = inheritedMappings.filter(mapping => {
+    // If there is no parent found, all the mappings should be exported.
+    if (parent == null || !StructureDefinitionProcessor.isProcessableStructureDefinition(parent))
+      return mappings;
```

Comment:
We don't actually have a linter rule for this, but I feel like we always use `{` `}` around the body of `if` statements (except, possibly, for one-liners).  Is that just me?  @mint-thompson?

Reply:
Ah I'm not sure if we have a rule, but that's something I totally support. This was a new line until prettier changed it on me and I didn't notice that it didn't add the `{ } `.

Reply:
Fixed in e0b28d5!